README.md

```diff
@@ -883,6 +883,88 @@ token.refresh()
     });
 ```
 
+### Collection Factory
+#### Search
+```js
+'use strict';
+const Model = require('screwdriver-models');
+const factory = Model.CollectionFactory.getInstance({
+    datastore
+});
+const config = {
+    params: {
+        userId: 12345
+    }
+};
+
+factory.list(config).then(collections => {
+    // Do stuff with list of collections
+});
+```
+
+| Parameter     | Type   | Description         |
+| :-------------| :------| :-------------------|
+| config        | Object | Config object       |
+| config.params | Object | Fields to search on |
+
+#### Create
+```js
+factory.create(config).then(model => {
+    // do stuff with collection model
+});
+```
+
+| Parameter          | Type   | Required | Description                                              |
+| :------------------| :------| :--------| :--------------------------------------------------------|
+| config             | Object | Yes      | Configuration Object                                     |
+| config.userId      | Number | Yes      | User that this collection belongs to                     |
+| config.name        | String | Yes      | Collection name                                          |
+| config.description | String | No       | Collection description                                   |
+| config.pipelineIds | Array  | No       | List of ids of pipelines associated with this collection |
+
+#### Get
+Get a collection based on unique id of collection.
```

Comment:
If userId+name also uniquely determine a collection, those can be passed in here too. See https://github.com/screwdriver-cd/models/pull/178/files#diff-04c6e90faac2675aa89e2176d2eec7d8R552